Even more fixes for the special cases of v18 textures

Fronkln opened the pull request.

Fronkln committed:
Even more fixes for the special cases of v18 textures

lewa-j commented:
Second condition will never happen.
You have code like
```
if(a){
...
}else{
if(a){
...
}
}
```

Fronkln commented:
Yep, magically it started causing issues again so i'm working on fixing it

Fronkln committed:
Checking only for embedded files & improvements 

This last change was tested on

city_test09.bsp
airex_start.bsp

Without any issues, conditions are confirmed working

Fronkln commented:
@lewa-j Take a look at this new code, just made sure it all works and all conditions are working as expected
